### ISSUE TITLE
check gradient from pytorch matches gradient equations from original ALCOVE paper

### ISSUE THREAD

**Issue by wkvong** [alexatartaglini/alcove#10]
lower priority, but a useful learning exercise!

go through the appendix that derives the learning rule for updating the weights in the ALCOVE model, and look at the equations that specify the derivatives/gradients for updating both the associative weights and the attentional weights. you'll want to convert these equations into python code and see if the results match the results produced by pytorch after a call of `loss.backward()`
